setup/hr_family/odoo/__init__.py

```diff
@@ -0,0 +1 @@
+__import__('pkg_resources').declare_namespace(__name__)
```

Comment:
@adrienpeiffer That is what i am talking about, if the directory is automatically generated, why spending time on generated it manually...but it's ok for me even like this...